notebooks/Zero_cost_Zero_time_Zero_shot_Financial_Sentiment_Analysis.ipynb

```diff
@@ -0,0 +1,2428 @@
+{
```

Comment:
<p>interesting, didn't know the WOS approach/metric. I suppose that word order is only one thing that transformers take into account. Another aspect would be semantic (dis)similarity of different strings. With a countvectorizer you only capture the exact words that are in the training corpus, but it can't capture semantically similar words that are outside the training data distribution</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/setfit/pull/497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>